inference-engine/tests_deprecated/unit/engines/vpu/myriad_tests/helpers/mock_icore.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Mocked core will be linked from `ieUnitTestUtils`. Is'n it? (`unit_test_utils\mocks\....hpp`)

Reply:
@iefode But this is deprecated tests..

Reply:
Do you mean, I need to link `unit_test_utils` from `tests` and `deprecated_tests`?

Reply:
You need to put mock for core to unit_test_utils and link here.

Reply:
`Unit_tests_utils` are linked to the deprecated unit tests...
![image](https://user-images.githubusercontent.com/22008923/87698241-e25faa00-c79b-11ea-9a9a-d3a10143365b.png)

Reply:
> You need to put mock for core to unit_test_utils  

It's already here. I need only remove `inference-engine/tests_deprecated/unit/engines/vpu/myriad_tests/helpers/mock_icore.h`

> Unit_tests_utils are linked to the deprecated unit test  

Ok, I get it. Thank you for the screenshot)